gbfs-validator/versions/schemas/v3.0-RC/vehicle_types.json

```
@@ -0,0 +1,268 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC/gbfs.md#vehicle_typesjson-added-in-v21-rc",
+  "description":
+    "Describes the types of vehicles that System operator has available for rent (added in v2.1-RC).",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description":
+        "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description":
+        "GBFS version number to which the feed conforms, according to the versioning framework.",
+      "type": "string",
+      "const": "3.0-RC"
```

Let's add v3.0 to v3.0-RC schemas the same way as in https://github.com/MobilityData/gbfs-json-schema/pull/88. Thank you!

```suggestion
      "enum": [
        "3.0-RC",
        "3.0"
      ]
```